docs/contents/pdf_model.rst

```diff
@@ -180,6 +223,8 @@ UML PDF model
         Figure -[#COL_DERIVE]-|> Element
         Chapter -[#COL_DERIVE]-|> Element
         Paragraph -[#COL_DERIVE]-|> Element
```

Comment:
is `Paragraph` still needed if we introduce `HorizontalBox`? I mean in your diagram a Paragraph can hold multiple boxes, but that is not correct. A `Paragraph` maps 1:1 to a `HorizontalBox`. So we could get rid also get rid of `Paragraph`. I like the idea.

Reply:
You are right. Paragraph can only hold one HorizontalBox. I think it would be a good idea to get rid of Paragraph, but how should we deal with Chapter? As LTTextBox is converted to either Chapter or Paragraph(HoriozntalBox). If Paragraph is removed, should the text of Chapter contain HorizontalBox?